.vscode/launch.json

```diff
@@ -124,7 +124,26 @@
       ],
       "stopOnEntry": false,
       "sourceMaps": true,
-      "outFiles": ["${workspaceRoot}/packages/*/out/**/*.js"],
+      "outFiles": [
+        "${workspaceRoot}/packages/*/out/**/*.js"
+      ],
+      "preLaunchTask": "Compile"
+    },
+    {
+      "name": "Launch Salesforce DX VS Code SLDS Linter Tests",
```

Comment:
@engai - This one probably needs to be updated since I removed the ${workspaceRoot}/packages/salesforcedx-slds-linter/assets/sfdx-simple